Svenbjorn/whatson
# made filters sticky

Monday link:
https://visitfaroeislands.monday.com/boards/4732496989/pulses/4562412004

TO TEST:
- Open whatson with this pull request
- Look at events, tours, places and hiking
- Ensure that: 
  - Header section is no longer sticky when scrolling down
  - Filters scroll to the top and remain there with sticky position
  - No other elements are affected adversely
